model/registry.go

```diff
@@ -107,8 +107,8 @@ type IstioRegistry struct {
 	ConfigRegistry
 }
 
-// RouteRules lists all rules in a namespace (or all rules if namespace is "")
-func (i *IstioRegistry) RouteRules(namespace string) []*proxyconfig.RouteRule {
+// ListRouteRules lists all rules in a namespace (or all rules if namespace is "")
+func (i *IstioRegistry) ListRouteRules(namespace string) []*proxyconfig.RouteRule {
 	out := make([]*proxyconfig.RouteRule, 0)
 	rs, err := i.List(RouteRule, namespace)
```

Comment:
Good point. Will do. Renaming is disruptive but we don't want to get stuck with bad names, too.